server/bleep/src/webserver/answer.rs

```diff
@@ -69,8 +69,10 @@ pub(super) async fn vote(
 pub struct Answer {
     pub q: String,
     pub repo_ref: RepoRef,
-    #[serde(default = "default_model")]
-    pub model: agent::model::AnswerModel,
+    #[serde(default = "default_answer_model")]
+    pub answer_model: agent::model::LLMModel,
+    #[serde(default = "default_agent_model")]
+    pub agent_model: agent::model::LLMModel,
```

Comment:
i believe this is a breaking change to the API. the FE currently this parameter in `model`, but the name has been changed to `answer_model`. here is what i think we should do:

* so we can either make the corresponding change on the FE here (cc @anastasiya1155)
* call it `answer_model` and use `#[serde(rename = "model")]`
* revert to old name   

side note: do we want to be able to configure `agent_model` with request parameter??

Reply:
I changed in the FE from model to answer_model (see comment in client/src/components/Chat/index.tsx)